fvtest/porttest/si.cpp

```diff
@@ -867,7 +867,10 @@ TEST(PortSysinfoTest, sysinfo_test_sysinfo_set_limit_CORE_FILE)
 		uint64_t originalHardLimit = 0;
 		uint64_t currentLimit = 0;
 		const uint64_t descriptorLimit = 256;
-
+	#if defined(OSX)
+	  outputErrorMessage(PORTTEST_ERROR_ARGS, "Skipping test: sysinfo_test_sysinfo_set_limit_FILE_DESCRIPTORS: resource limits unsupported on macOS, see issue #1234 for more details\n");
+	  return;
+	#else
```

Comment:
Indentation of preprocessor directives should start at column 0 to be consistent with the rest of the codebase. We also need to add the `/* defined(OSX) */` comments for the `#else` and `#endif` as per @rwy0717's suggestion [1].

This `#ifdef` should be right after the `OMRPORT_ACCESS_FROM_OMRPORT` line too. No need to initialize the variables if we are skipping the test. Also we seem to reference issue 1234 here, which looks like a placeholder. We'll need to open up a proper issue describing the problem and reference the correct issue number here. Let me work on opening up that issue now.

I also don't think we need the `return;` here as it serves no purpose.

[1] https://eclipse-omr.slack.com/archives/C010F8XPRT9/p1585080941104000